pytest.ini

```diff
@@ -5,4 +5,5 @@ markers =
    slow: Tests that take long time to run locally. Skipped by 'make test-fast'.
 # By default, treat warnings as errors
 filterwarnings =
+    ignore::ResourceWarning
```

Comment:
**issue**: IIUC this ignores ResourceWarning-type warning for all of our tests. Ignoring warnings should be the last resort, and even if we do it, it should be as narrow as possible. Can you move it to the test case that requires it?